app/models/concerns/project_media_getters.rb

```diff
@@ -213,4 +213,10 @@ def team_avatar
   def fact_check
     self.claim_description&.fact_check
   end
+
+  def explainers_title
```

Comment:
Since this returns the concatenated titles for all explainers, I suggest to call this method `explainers_titles`.